examples/Atlas/sdf/random_terrain.world

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we actually want to include this in the repository? If this is just a sample output from `generateRandomBoxWorld.m` then I think we probably don't need to include it.